bitnami/orangehrm/templates/NOTES.txt

```diff
@@ -64,7 +64,7 @@ host. To configure OrangeHRM to use and external database host:
 {{- $passwordValidationErrors := list -}}
 {{- $secretName := include "orangehrm.secretName" . -}}
 
-{{- if not .Values.existingSecret -}}
+{{- if and (not .Values.existingSecret) (not (include "common.secrets.exists" (dict "secret" $secretName "context" $))) -}}
```

Comment:
This will be a very common statement for all the asset, could we find a way of moving it to the common helpers? Maybe in "common.validations.values.single.empty" or "common.errors.upgrade.passwords.empty"

Reply:
I have tried a couple of things but I haven't been able to change neither of those helpers without changing their current behaviour. If we decide to have `customValue` in singular and as a String, I think it would be easy to move the entire validation of passwords to `common`.

Reply:
do you mean to add the validation inside "common.secrets.passwords.manage"?

Reply:
Could we test it? let's see it, I think that could be really good, don't you think?